code/modules/hydroponics/grown/towercap.dm

```diff
@@ -162,6 +165,7 @@
 
 /obj/structure/bonfire/Crossed(atom/movable/AM, oldloc)
 	if(burning)
+		add_attack_logs(src, AM, "burned by a bonfire (Lit by [lighter])", ATKLOG_ALMOSTALL)
```

Comment:
Checking `add_attack_logs` the proc: This will send a message to admins (who have the almostALL logging setting) even when say a ckeyless mob gets burned. Or just a locker.
Might want to do an istype check here.... that or rework `create_attack_logs` since that one also needs a rework

Reply:
Seconded, the logging level should depend on whether what is being burned is a mob/player or not